schemas/com.snowplowanalytics.snowplow.badrows/snowflake_specific_failure/jsonschema/1-0-0

```diff
@@ -0,0 +1,561 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Snowflake Loader specific bad row",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "snowflake_specific_failure",
```

Comment:
If we remove `specific` part, will `snowflake_failure` mean that failure is not specific to snowflake? Also, the difference between "failure" and "error" is subtle, but this one is certainly not "failure".

Reply:
You are right, `specific` part can be removed. Then, is `snowflake_error` okay ?

Reply:
Yes, fine by me.